str/indel_locus_extractor.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+# pylint: disable=missing-function-docstring,no-member
+"""
+This script pulls out the locus coordinates of every indel recorded in an existing Hail Matrix Table.
+
+ analysis-runner --dataset "tob-wgs" \
+    --description "indel coordinates extractor" \
+    --access-level "test" \
+    --output-dir "hoptan-str/indel-experiment/" \
+    indel_locus_extractor.py --file-path=gs://cpg-tob-wgs-test/mt/v7.mt
+
+"""
+
+import hail as hl
+import click
+
+from cpg_utils.config import get_config
+from cpg_utils.hail_batch import output_path, init_batch
+from cpg_workflows.batch import get_batch
+
+
+config = get_config()
```

Comment:
```suggestion
import hail as hl
import click

from cpg_utils.hail_batch import output_path, init_batch
from cpg_workflows.batch import get_batch

```

If you don't create a nested batch job you don't need the config here